src/OpenSage.Game/Logic/Object/GameObjectCollection.cs

```diff
@@ -88,6 +89,16 @@ public bool TryGetObjectByName(string name, out GameObject gameObject)
             return _nameLookup.TryGetValue(name, out gameObject);
         }
 
+        public List<GameObject> GetObjectsByName(string typeName)
```

Comment:
I don't see that this method is used anywhere?

Reply:
removed